src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -167,9 +171,35 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
       exportAllOptions: [],
       exportAllSelectedOptions: {},
       isIncludeReferencesDeepChecked: true,
+      currentWorkspaceId: this.props.workspaces.currentWorkspaceId$.getValue(),
+      workspaceEnabled: this.props.applications.capabilities.workspaces.enabled,
     };
   }
 
+  private get workspaceIdQuery() {
+    const { currentWorkspaceId, workspaceEnabled } = this.state;
```

Comment:
> when do we set the state? looks like we get the workspace enabled and current workspace id from the prop?

@BionIT Correct, in this PR these two states won't be set so adjust the code just consume it from props.